### DISCUSSION TITLE
InDesign Script to resize, rotate, and center images on specific pages

### DISCUSSION THREAD

**Discussion by LeFinneus** [rsms/inter#617] [General]
A newbie once transgressed in this space, mistakenly and was sharply and summarily crucified by user @kenmcd 

The crime has been committed. I apologized and did my best to delete the content.

Maybe now he can sleep tonight.

Yeesh.

**Comment by kenmcd** [comment]
Did you accidentally post this in the wrong repository?
Because this has nothing to do with the Inter fonts.
And is entirely inappropriate to post here.
You should delete this post.

**Reply by LeFinneus** [reply]
There's no need to be rude. Yes, it was a mistake posting it here. I thought I was posting on a JS board. How many you ever forgive me?
